polaris/ocean/tasks/cosine_bell/analysis.py

```diff
@@ -44,135 +43,64 @@ def __init__(self, component, resolutions, icosahedral, subdir,
-
-        section = self.config['cosine_bell']
-        if self.icosahedral:
-            conv_thresh = section.getfloat('icos_conv_thresh')
-            conv_max = section.getfloat('icos_conv_max')
-        else:
-            conv_thresh = section.getfloat('qu_conv_thresh')
-            conv_max = section.getfloat('qu_conv_max')
-
-        if conv < conv_thresh:
-            raise ValueError(f'order of convergence '
-                             f' {conv} < min tolerence {conv_thresh}')
-
-        if conv > conv_max:
-            warnings.warn(f'order of convergence '
-                          f'{conv} > max tolerence {conv_max}')
-
-    def rmse(self, mesh_name):
-        """
-        Compute the RMSE for a given resolution
+        Get the exact solution
 
         Parameters
         ----------
-        mesh_name : str
-            The name of the mesh
+        field_name : str
+            The name of the variable of which we evaluate convergence
+            For the default method, we use the same convergence rate for all
+            fields
+        time: float
+            The time at which to evaluate the exact solution in seconds.
+            For the default method, we always use the initial state.
 
         Returns
         -------
-        rmseValue : float
-            The root-mean-squared error
-
-        nCells : int
-            The number of cells in the mesh
+        solution: np.ndarray of type float
+            The minimum convergence rate
         """
 
+        if field_name != 'tracer1':
+            print(f'Variable {field_name} not available as an analytic '
+                  'solution for the cosine_bell test case')
+
         config = self.config
         latCent = config.getfloat('cosine_bell', 'lat_center')
         lonCent = config.getfloat('cosine_bell', 'lon_center')
         radius = config.getfloat('cosine_bell', 'radius')
         psi0 = config.getfloat('cosine_bell', 'psi0')
-        convergence_eval_time = config.getfloat('spherical_convergence',
-                                                'convergence_eval_time')
-
         ds_mesh = xr.open_dataset(f'{mesh_name}_mesh.nc')
         ds_init = xr.open_dataset(f'{mesh_name}_init.nc')
-        # find time since the beginning of run
-        ds = xr.open_dataset(f'{mesh_name}_output.nc')
-        for j in range(len(ds.xtime)):
-            tt = str(ds.xtime[j].values)
-            tt.rfind('_')
-            DY = float(tt[10:12]) - 1
-            if DY == convergence_eval_time:
-                sliceTime = j
-                break
-        HR = float(tt[13:15])
-        MN = float(tt[16:18])
-        t = 86400.0 * DY + HR * 3600. + MN
         # find new location of blob center
         # center is based on equatorial velocity
         R = ds_mesh.sphere_radius
-        distTrav = 2.0 * 3.14159265 * R / (86400.0 * convergence_eval_time) * t
+        distTrav = 2.0 * np.pi * R / time
         # distance in radians is
         distRad = distTrav / R
```

Comment:
I think this isn't correct, in part because the original was a mess.  I think we need to add:
```
vel_pd = config.getfloat('cosine_bell', 'vel_pd')
```
up above and then this should just be:
```suggestion
        # distance in radians is
        distRad = 2.0 * np.pi * time / (86400.0 * vel_pd)
```
The original used `t` in the numerator and `convergence_eval_time` in the denominator, but the latter isn't really what we want if `convergence_eval_time != vel_pd`.